app.go

```diff
@@ -892,10 +892,10 @@ func (app *App) Hooks() *Hooks {
 }
 
 // Test is used for internal debugging by passing a *http.Request.
-// Timeout is optional and defaults to 1s, -1 will disable it completely.
+// Timeout is optional and defaults to 10s, -1 will disable it completely.
 func (app *App) Test(req *http.Request, msTimeout ...int) (*http.Response, error) {
 	// Set timeout
-	timeout := 1000
+	timeout := int((10 * time.Second) / time.Millisecond)
```

Comment:
True. I don't really know how to avoid it though. As most of the tests run concurrently now, individual tests might take longer than the previous 1 second defined here due to the Go scheduler rescheduling execution between tests.

What do you think of adding and using a new `app.TestWithDynamicTimeout(req *http.Request, msTimeout ...int)` which defaults this to a timeout of 10s as of now, and might be further increased in the future?

Reply:
@ReneWerner87 what do you recommend to do about this?